spec/models/claims/state_machine_spec.rb

```diff
@@ -400,17 +400,40 @@
   end
 
   describe 'refuse!' do
-    before { claim.submit!; claim.allocate!; claim.refuse!(reason_code: reason_codes) }
     let(:reason_codes) { ['wrong_ia'] }
     let(:last_state_transition) { claim.last_state_transition }
 
-    context 'claim state transitions (audit trail)' do
-      it 'updates #to' do
-        expect(last_state_transition.to).to eq('refused')
+    context 'when refused on first assessment' do
+      before { claim.submit!; claim.allocate!; claim.refuse!(reason_code: reason_codes) }
```

Comment:
**[not a blocker/nit pick]** Avoid multiple inline statements with `;`, specs are supposed to be expressive anyway, so a couple . Also consider following a similar approach to [comment](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2384/files#r191457874) whereas in this case using the positive expectation:

```ruby
expect { claim.refuse!(reason_code: reason_codes) }.to change { claim.assessment.total }.to(0)
```